github.go

```diff
@@ -157,3 +208,45 @@ func (g *GitHubPullRequest) comment() ([]*github.PullRequestComment, error) {
 	}
 	return comments, nil
 }
+
+// ---
+// GitHub PullRequest Review API Implementation
+// ref: https://github.com/google/go-github/issues/495
+
+const (
+	mediaTypePullRequestReview = "application/vnd.github.black-cat-preview+json"
+)
+
+type Review struct {
```

Comment:
it's ok because this struct is temporary one and replace it with go-github implementation later.